_config.yml

```diff
@@ -18,33 +18,39 @@
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
 
-title: NCCS
+title: National Center for Charitable Statistics
+short_title: NCCS
```

Comment:
The short title was added to help prevent nav collision if many items are added to the navigation list.